app.rb

```diff
@@ -0,0 +1,171 @@
+require_relative 'person'
+require_relative 'book'
+require_relative 'student'
+require_relative 'classroom'
+require_relative 'rental'
+require_relative 'teacher'
+
+class App
+  def initialize
+    @books = []
+    @students = []
+    @teachers = []
+    @people = []
+    @rentals = []
+  end
+
+  # list of books
+  def list_books
+    if @books.empty?
+      puts 'No books found.'
+    else
+      @books.each_with_index do |book, index|
+        puts "#{index}) Title: \"#{book.title}\", Author: #{book.author}"
+      end
+    end
+    nil
+  end
+
+  # list of people method
+  def list_people
+    print 'Do you want to Create a list of a students (1) or a teachers (2)? [Input the number]: '
+    list_person = gets.chomp.to_i
+    case list_person
+    when 1
+      list_student
+    when 2
+      list_teacher
+    else
+      puts "\nInvalid input!!\n\n"
+    end
```

Comment:
- [ ] when the option to list people is selected it doesn't show the way it should as done in this [reference video](https://www.youtube.com/watch?v=vkkgrhD6aXQ), please kindly make the necessary changes to show all people rather than first asking them to select :+1: 
- project requirement
![op-2](https://github.com/mohamedgamaleldin999999/School-library/assets/89055502/cc3840b6-1a67-4236-a00e-522cca4bb5fa)